source/kraken/tests/realtime_test.cpp

```diff
@@ -2740,3 +2740,380 @@ BOOST_AUTO_TEST_CASE(should_get_correct_base_stop_time_with_lollipop_II) {
     BOOST_CHECK_EQUAL(stoptime_rltm_B_bis.get_base_stop_time(), &vj->stop_time_list[3]);
     BOOST_CHECK_EQUAL(stoptime_rltm_D.get_base_stop_time(), &vj->stop_time_list[4]);
 }
+
+BOOST_AUTO_TEST_CASE(add_new_trip) {
```

Comment:
If it's possible it would be better to split, using `BOOST_FIXTURE_TEST_CASE`.

Minor: maybe even in a new file.

Reply:
For tests strongness, I need to follow cases, this is the reason why it is a long long test. thanks to that, we can monitor all counter example
```
BOOST_REQUIRE_EQUAL(pt_data.vehicle_journeys.size(), 1);
// Do tests
BOOST_REQUIRE_EQUAL(pt_data.vehicle_journeys.size(), 2);
etc
```

Reply:
Suggested some places to separate if you think it's worth it.
I resolve this feed but use that boost fixture if you split tests ;)